backend/src/test/java/quarano/actions/web/ActionItemsControllerWebIntegrationTests.java

```diff
@@ -75,9 +75,6 @@ void rendersAnomaly() throws Exception {
 		var document = JsonPath.parse(response);
 
 		assertThat(document.read("$.comments", JSONArray.class)).isEmpty();
-		assertThat(document.read("$.anomalies.health", JSONArray.class)).hasSize(1);
```

Comment:
If that change is triggered by the test person used here is an index case person then I'd much rather switch to a contact case person than dropping the assertions. Otherwise we "lose" the test for the contact cases rendering the anomalies.

Reply:
ok, i will try to sacrifice the assertion ;). I did exactly this with index case gustav in another test case already.

Reply:
If you test this function in another test case, then the whole method could go away. However, I do not see a corresponding new method.